Formula/libmf.rb

```diff
@@ -0,0 +1,41 @@
+class Libmf < Formula
+  desc "open source tool for approximating an incomplete matrix."
+  homepage "https://www.csie.ntu.edu.tw/~cjlin/libmf/"
+  url "http://www.csie.ntu.edu.tw/~cjlin/libmf/libmf-2.01.zip"
+  sha256 "75bb98a4e8f1a318d7d07556583727b4e301759904570bf527972d964d34ef30"
+  patch :DATA
```

Comment:
I don't think I can submit this patch to the upstream developers. This patch disables OpenMP which is not supported by clang and replace a tag which is not supported for clang linker, so it is a platform dependent patch.

Do you think it is necessary that I add some comment to explain the patch in the formula?